README.md

```diff
@@ -1,37 +1,161 @@
-# Local startup
+# Verify Local startup
 
-Repository with useful scripts for creating PKI, building and starting Verify apps
+This repositories primary purpose is to allow anyone the ability to build and run the Verify Hub and related microservices.  It makes extensive use of Git, Docker and Ruby to get the job done.
+
+This repository also provides a number of scripts for generating a new Public Key Infrastructure (PKI) used by the verify applications.
+
+## Contents
+
+* [Before you start](#before-you-start)
+* [Getting Started](#getting-started)
+* [Stopping the Hub](#stopping-the-hub)
+* [Startup Commandline Options](#Startup-Commandline-Options)
+* [Useful Scripts](#useful-scripts)
+    * [XMLSecTool in Docker](#XMLSecTool-in-Docker)
+    * [Generate PKI](#Generate-PKI)
+    * [Create environment for local running](#Create-environment-for-local-running)
+* [Support and raising issues](#Support-and-raising-issues)
+* [Licence](#Licence)
 
 ## Before you start
 
-You'll need to install [cfssl](https://github.com/cloudflare/cfssl) which is used to generate the PKI.
+You'll need a computer with at least 8GB (for linux systems) or 16GB (for Mac and Windows systems) and at least a 4 core processor.  You'll also need administrator privilages in order to installed the required components listed below.
+
+You will need to install Git, [Docker](https://www.docker.com/get-started) and Ruby 2.7.2.  If you have [rbenv](https://github.com/rbenv/rbenv) installed the script will automatically pull and compile the correct version of ruby for you.
+
+Ideally if you are on a Linux based system you'll also need to add yourself to the docker group:
+
+```sh
+sudo usermod -a -G docker $(whoami)
+```
+
+### Notes for Mac Systems
+
+Docker does not run natively on Mac OS systems and instead uses a VirtualBox Virtual Machine which is controlled by the DockerApp.  Before you start you need to allocate at least 6GB of RAM (Ideally 8GB) to this virtual machine.  Additionally if you increase the number of CPUs allocated from the default of 2 you can improve the build performance.  Details can be found in the [Docker User Manual](https://docs.docker.com/docker-for-mac/#resources).
+
+## Getting Started
+
+To run the hub locally first you need to clone this repository to an empty directory.
+
+```sh
+mkdir Verify
+cd Verify
+git clone https://github.com/alphagov/verify-local-startup.git
+cd verify-local-startup
+```
+
+Now you're in the `verify-local-startup` directory running the hub should be as simple as:
+
+```sh
+./startup.sh
+```
+
```

Comment:
Perhaps you could add a sentence to explain that the script will fetch the public repositories it needs (and which repositories they are).

Reply:
I just wonder if it is necessary to put a disclaimer this doesn't work on windows

Reply:
@jkunle But it does work on Windows... If you have evidence to say it doesn't please file a bug.  It runs just find under Docker for Windows and WSL.  If we wanted a better experience I guess we could write a PowerShell script to sit along side the bash script but I am not sure I see usefulness in that.